provider-tutorials/provider-cli.md

```diff
@@ -210,3 +210,22 @@ For `--network`you have two options, either `mainnet` or `rinkeby`. For `--to-ad
 
 **Note that if you decided to use an external wallet during your setup process, you can connect to ZkSync's wallet at** [**https://wallet.zksync.io/**](https://wallet.zksync.io/) **and exit that way. In the scenario that a different payment driver is being used, you will need to use the relevant available options to connect and access your tokens.**
 
+## Advanced Settings
+
+`ya-provider` allows to fine-tune config created with `golemsp settings` using commands `config`, `preset`, `profile`, and `exe-unit`.
+
+Additionally, it allows configuring certificate _keystore_ and domain _whitelist_.
```

Comment:
I'm unsure if it's necessary to accent these words ... plus, there's a repetition... and generally, maybe just a bit more context would be due... 

```suggestion
Additionally, it enables configuration of the certificate keystore and of the domain whitelist, which are used to control what kind of outbound traffic is allowed out of the VM containers run by the requestors on your machine.
```